sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -448,6 +453,25 @@ public AutoConfiguredOpenTelemetrySdk build() {
 
       if (sdkEnabled) {
         SdkMeterProviderBuilder meterProviderBuilder = SdkMeterProvider.builder();
+        if (config.getBoolean("otel.instrumentation.http.server.add-server-attributes", false)) {
```

Comment:
I don't think this belongs in our core SDK autoconfigure. This looks to be an instrumentation-specific option. Let's talk about the right place for this configuration in a SIG meeting.